source/extensions/extensions_build_config.bzl

```diff
@@ -276,3 +276,31 @@ EXTENSIONS = {
 # need to directly reference Envoy extensions.
 EXTENSION_CONFIG_VISIBILITY = ["//:extension_config"]
 EXTENSION_PACKAGE_VISIBILITY = ["//:extension_library"]
+
+# Extensions with unknown security posture. This will be kept in sync by CI.
+# This is used to exclude while fuzzing.
+UNKNOWN_SECURITY_EXTENSIONS = [
+    "envoy.access_loggers.wasm",
```

Comment:
Can we do this differently? In the fuzz rule, can we make use of the security posture information somehow and nop it? Just worried about maintainability and single SoT.

Reply:
Ah, right it's checked validated in CI against security posture info.

I get that, I wasn't sure at all how to reflect back on the status information from the extensions, but mostly want to block CI asap. If you have a Bazel docs link (or example?) please link!

Reply:
How about having https://github.com/envoyproxy/envoy/blob/a809844ddeda2fce1f5d3cdfe6204fafb7fd4452/bazel/envoy_library.bzl#L115 add to the binary some static registration of security posture?

I think we want this anyway, since when we add the "secure profile", we want to refuse to initialize if any insecure extensions are linked. LMK if this seems too much time given the desire to land something soon, but it seems the most robust for future needs.